database/changelog.xml

```diff
@@ -14,4 +14,6 @@
     <include file="changesets/20200316_ExternalWholeGenomeReprocessingEnum.xml"  relativeToChangelogFile="true"/>
     <include file="changesets/20200317_ExternalWholeGenomeReprocessingTable.xml" relativeToChangelogFile="true"/>
     <include file="changesets/20200424_RenameLoadColumnToItems.xml"              relativeToChangelogFile="true"/>
+    <include file="changesets/20200600_CopyfileEnum.xml"                         relativeToChangelogFile="true"/>
+    <include file="changesets/20200601_CopyfileTable.xml"                        relativeToChangelogFile="true"/>
```

Comment:
Not sure if you already know this, but you can use `<includeAll>` to load all changelogs in a directory. It gives you less control but it makes adding changelogs easier, since usually what you want is to load all changelogs.

Reply:
Nice. This'll reduce noise in future pull requests

Reply:
On reflection, the liquibase doc actually suggests advises against it:
https://docs.liquibase.com/concepts/advanced/includeall.html